_data/nav.yml

```diff
@@ -234,6 +234,9 @@
       - url: "#API"
     - url: "advcode"
     - url: "advtable"
+      pages:
+      - url: "#Events"
```

Comment:
Should this be `#advtable_events` to match with the subpages of the other plugins?

Reply:
no. heading name is used. If our system could handle more structure, this would be common.